src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -271,7 +271,7 @@ protected virtual JToken GetUnclaimedGas(JArray _params)
                 script_hash = null;
             }
             if (script_hash == null)
-                throw new RpcException(-100, "Invalid address");
+                throw new RpcException(RpcError.InvalidParams);
```

Comment:
L237: we need to check if the specified session exists and if not then throw `UnknownSession` RPC error.

Reply:
Resolved.